### PR TITLE
Vishisht cd 1.0

### PR DESCRIPTION
**Pull request by vishisht-dubey**

### CODE REVIEW

File: .env
Comment (Veer0x1) [inline .env:1]: gitignore krdo isse

File: context/AuthContext.jsx
Comment (Veer0x1) [inline context/AuthContext.jsx:1]: Have you checked signin with Popup is working, afaik last time ye kam nahi kr raha the

File: context/AuthContext.jsx [hunk 1]
```diff
@@ -71,19 +75,60 @@ export const AuthContextProvider = ({ children }) => {
       }
     }
   };
+  console.log(user);
```
Comment (Veer0x1) [inline context/AuthContext.jsx]: console log hata dena bhai, test krne k bad.

File: utility/firebase.js
Comment (Veer0x1) [inline utility/firebase.js:1]: environment variable use kro bhai, taki deployment sirf env file hi change krni rahegi

**Comment by vishisht-dubey** [comment]
@Veer0x1 dont merge this pr now i will raise with improved fixes

**Pull request closed without merging**